docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -15,9 +15,8 @@ The minimal required version of .NET Framework is 4.7.2.
 
 ## Developing with Visual Studio 2022
 
-1. [Visual Studio 2022 17.0 Preview](https://visualstudio.microsoft.com/vs/preview/vs2022/)
+1. [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/vs2022/)
```

Comment:
Should we add the word "Current" in there some place? or does it speak for itself?

Reply:
```suggestion
1. Use latest [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/vs2022/)
```